docs/index.js.html

```diff
@@ -122,52 +132,69 @@ <h1 class="page-title">Source: index.js</h1>
   }
 
   getSegments() {
-    if (this.screenSpanning === "none") {
+    if (this.verticalViewportSegments === 1 &amp;&amp; this.horizontalViewportSegments === 1) {
       return [
         { left: 0, top: 0, width: window.innerWidth, height: window.innerHeight },
       ];
     }
-
+    let segments = [];
     // The fold is defined as a segment here because it's used in the css spaning polyfill.
-    if (this.screenSpanning === "single-fold-horizontal") {
-      const screenCenter = (window.innerHeight - this.browserShellSize) / 2;
+    if (this.horizontalViewportSegments > 1) {
+      let numberOfFolds = this.horizontalViewportSegments - 1;
+      const availableHeight = window.innerHeight - this.browserShellSize;
+      let topOffset = 0;
       const width = window.innerWidth;
-      return [
-        { top: 0, left: 0, width, height: screenCenter - this.foldSize / 2 },
-        { top: screenCenter - this.foldSize / 2, height: this.foldSize, left: 0, width },
-        { top: screenCenter + this.foldSize / 2, left: 0, width, height: screenCenter - this.foldSize / 2 }
-      ];
+      const height = availableHeight / this.horizontalViewportSegments - this.foldSize *
+        numberOfFolds / this.horizontalViewportSegments;
+      for (let i = 0; i &lt; this.horizontalViewportSegments + numberOfFolds; ++i) {
+        if (i % 2 === 0)
+          segments[i] = { top: topOffset, left: 0, width: width, height: height };
+        else
```

Comment:
big mistake here :) no {} and last line is then outside of the if sentence... please always use {}